addon/TheUndermineJournal.toc

```diff
@@ -2,7 +2,7 @@
 ## Title: The Undermine Journal
 ## Notes: %s
 ## OptionalDeps: Auctionator, AuctionLite, LibExtraTip
-## SavedVariablesPerCharacter: TUJTooltipsHidden
+## SavedVariablesPerCharacter: TUJTooltipsSettings
```

Comment:
Changed the name of the saved variables to the new table that I used for the additional saved settings.